samples/animetrix-react-app/src/components/AppRouter.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { createBrowserRouter, RouterProvider, Navigate } from "react-router-dom";
+import { AuthenticatedComponent } from "@asgardeo/auth-react";
+import ProtectedRoute from "./ProtectedRoute";
+import { LandingPage } from "../pages/LandingPage";
+import { HomePage } from "../pages/HomePage";
+import { InsightsPage } from "../pages/InsightsPage";
+import { Unauthorized } from "../pages/UnauthorizedPage";
+
+const AppRouter = ({ currentUserRoles }) => {
+    const router = createBrowserRouter([
```

Comment:
Shall we move this outside of the component to avoid getting recreated on every render?